packages/resizin/src/upload.ts

```diff
@@ -0,0 +1,48 @@
+import fetch from 'isomorphic-fetch';
+import NodeFormData from 'form-data';
+import Promise from 'promise-polyfill';
+
+export const uploadImage = (url?: string, api_key?: string, id?: string, file?: string) => {
```

Comment:
hezka funkcicka haha :))

Reply:
ty volitelný parametry tady jsou vlastně jen kvuli testům, ale pak jsem přišel na to jak to udělat i bez toho, upravim 👍